gulpfile.js

```diff
@@ -17,8 +17,11 @@ const commonjs = require("@rollup/plugin-commonjs");
 const nodeResolve = require("@rollup/plugin-node-resolve").nodeResolve;
 const typescript = require("@rollup/plugin-typescript");
 const { exec } = require("child_process");
-const setProductionEnv = require("./processor").setProductionEnv;
-const updateComponentTemplateUrl = require("./processor").updateComponentTemplateUrl;
+const {
+	setProductionEnv,
```

Comment:
you formatting using tabs? so spaacey 😂

Reply:
oh wow this looks so bad... I wish Microsoft would do what they did in VSCode and just switch it to depend on the "tab size" setting, it looks so much nicer there 😭